manifests/s/StellarDataRecovery/StellarExtractorforWindowsBackup/6.0.0.0/StellarDataRecovery.StellarExtractorforWindowsBackup.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: StellarDataRecovery.StellarExtractorforWindowsBackup
+PackageVersion: 6.0.0.0
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://cloud.stellarinfo.com/StellarExtractorforWindowsBackup.exe
+  InstallerSha256: F88DD1A37A572ABD4CF2DB11C8CBACF04BB187BA9649515127BF5EC7DA6404DD
+  InstallModes:
+  - verysilent

```

```suggestion
  - silent
```
Reverts https://github.com/microsoft/winget-pkgs/pull/56009/commits/37c784b139c97e00163da736cbff66fedcd70140.

It should install in /verysilent when you pass in `--silent` to `winget install`.